docs-ui/components/docsLinks.tsx

```diff
@@ -129,3 +133,9 @@ const IconWrap = styled('div')`
     transform: translateX(${space(0.5)});
   }
 `;
+
+const Desc = styled('p')`
+  margin-top: ${space(0.5)};
+  font-size: 0.875rem;
+  color: ${p => p.theme.gray300};
```

Comment:
I don't remember does storybook have a dark mode toggle? And will this look OK if so?

Reply:
It does, and this and other global components look terrible in dark mode 😬

<img width="790" alt="Screen Shot 2021-09-23 at 4 08 49 PM" src="https://user-images.githubusercontent.com/44172267/134595445-f793b7c6-a4a5-498d-a3d8-f694a50108ba.png">

I'm aware of this, and think that it's a separate issue that should be dealt with soon. The dark mode plugin was designed for the stories tab, not the docs tab, which is where these global components will be. Ideally the whole docs tab should be in dark mode, but as far as I know there isn't any obvious solution to do so.